.pre-commit-config.yaml

```diff
@@ -44,12 +43,12 @@ repos:
     sha: v1.11.0
```

Comment:
might as well take this to 1.16.1 too -- there's some new py3 goodies in the later versions